challenges/objects-arrays.js

```diff
@@ -13,22 +13,48 @@
 // velociraptor, carnivorous, 15kg, 1.8m, Late Cretaceious
```

Comment:
great job on creating an array of objects, however this only required you to create 3 separate objects